.gitignore

```diff
@@ -129,3 +129,4 @@ node_modules/
 
 /src/assets/**/js/*.js
 /src/assets/**/js/*.LICENSE.txt
+docker-compose.yml
```

Comment:
Nope ... this file is required so can't ignore it...

Reply:
Alright noted, thank you will remove it from git ignore and push